TicTacToe2/TicTacToe/Player.cs

```diff
@@ -13,13 +13,14 @@ public Player(char symbol)
             Symbol = symbol;
         }
 
-        public void GetMove(out int x, out int y)
+        public void GetMove(out Math.Vec2 move)
```

Comment:
This function is now only returning a single value, so it would be preferable to have the function have a return type of Vec2 rather than using an out parameter.